ci/Dockerfile.ci-utils

```diff
@@ -1,12 +1,7 @@
 ARG BASE_IMAGE={{ base_image.image }}
 FROM $BASE_IMAGE AS base
 
-# source: https://cloud.google.com/storage/docs/gsutil_install#linux
-RUN curl --remote-name https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-447.0.0-linux-x86_64.tar.gz && \
-    tar -xf google-cloud-sdk-447.0.0-linux-x86_64.tar.gz && \
-    curl --remote-name https://dl.k8s.io/release/v1.21.14/bin/linux/amd64/kubectl && \
-    install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
-ENV PATH $PATH:/google-cloud-sdk/bin
+RUN hail-apt-get-install google-cloud-cli kubectl
 
 # https://bugs.mysql.com/bug.php?id=105288&thanks=sub
 RUN hail-apt-get-install xz-utils libncurses5 git && \
```

Comment:
Maybe combine these two to reduce the number of layers?

Reply:
done